src/util/tenor-utils.ts

```diff
@@ -130,85 +130,87 @@ type tenorMediaObject = {
 
 //
 
-const DEFAULT_RESPONSE_LIMIT = 100;
-const TWELVE_HOURS = 43_200_000;
+const DEFAULT_RESPONSE_LIMIT: number = 50;
+const TWELVE_HOURS: number = 43_200_000;
 
 // Exported Singleton class that will later be used in the program.
 
 export class TenorSingleton {
     private static instance: TenorSingleton | null | undefined;
     private client: axios.AxiosInstance;
-    private static cache: Map<string, string[]>;
+    private readonly cache: Map<string, string[]>;
 
     private constructor() {
         this.client = axios.create({
             baseURL: "https://tenor.googleapis.com/v2",
         });
-        return this;
+        this.cache = new Map();
+
+        setInterval(() => {
+            this.cache.clear();
+        }, TWELVE_HOURS);
     }
 
     public static getInstance(): TenorSingleton {
         if (!this.instance) {
             this.instance = new TenorSingleton();
-            this.cache = new Map();
-            setInterval(() => {
-                this.cache.clear();
-            }, TWELVE_HOURS);
         }
+
         return this.instance;
     }
```

Comment:
I also cleaned up the `getInstance` function by creating the interval in the constructor.